packages/components/src/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export type TextAreaProps = {
+  className?: string;
+  placeholder?: string;
+  rows?: number;
```

Comment:
I think it should have `value`, `onChange`, and `disabled` props as well.
In general, we control forms by using value and onChange props in a react app.
ref: https://reactjs.org/docs/forms.html#the-textarea-tag

Reply:
Sure, good note!

Added also readOnly, onBlur and defaultValue.